### PR TITLE
Fix: Actually use constants defined in interfaces

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] actually uses the constants as defined in the interfaces *everywhere*  

Follows #25.

### CODE REVIEW

File: test/Component/Video/RestrictionTest.php [hunk 1]
```diff
@@ -17,17 +17,11 @@ class RestrictionTest extends \PHPUnit_Framework_TestCase
 {
     use GeneratorTrait;
 
-    public function testConstants()
-    {
-        $this->assertSame('allow', Restriction::RELATIONSHIP_ALLOW);
-        $this->assertSame('deny', Restriction::RELATIONSHIP_DENY);
-    }
```
Comment (localheinz) [inline test/Component/Video/RestrictionTest.php]: @jingjingdong 

I forgot to *remove* this earlier, test exist [here](https://github.com/refinery29/sitemap/blob/master/test/Component/Video/RestrictionInterfaceTest.php#L18-L22) now!

File: test/Component/Video/VideoTest.php [hunk 2]
```diff
@@ -25,29 +25,6 @@ class VideoTest extends \PHPUnit_Framework_TestCase
...
-
-        $this->assertSame('yes', Video::REQUIRES_SUBSCRIPTION_YES);
-        $this->assertSame('no', Video::REQUIRES_SUBSCRIPTION_NO);
-
-        $this->assertSame('no', Video::FAMILY_FRIENDLY_NO);
-
-        $this->assertSame(256, Video::CATEGORY_MAX_LENGTH);
-
-        $this->assertSame(32, Video::TAG_MAX_COUNT);
-    }
```
Comment (localheinz) [inline test/Component/Video/VideoTest.php]: @jingjingdong 

I forgot to *remove* this earlier, test exist [here](https://github.com/refinery29/sitemap/blob/master/test/Component/Video/VideoInterfaceTest.php#L15-L36) now!

**Comment by jingjingdong** [comment]
:+1: Looking good for me!

**Comment by localheinz** [comment]
Thank you, @jingjingdong!

**Pull request merged**